x-pack/plugins/endpoint/common/models/event.ts

```diff
@@ -22,10 +22,33 @@ export function eventTimestamp(
   }
 }
 
+/** TODO, seems wrong */
 export function eventName(event: EndpointEvent | LegacyEndpointEvent): string {
   if (isLegacyEvent(event)) {
     return event.endgame.process_name ? event.endgame.process_name : '';
   } else {
     return event.process.name;
   }
 }
+
+export function eventId(event: ResolverEvent) {
```

Comment:
Would you mind moving these methods that take "ResolverEvent" to a file called "resolver_event"